lib/local-links-manager/distributed_lock.rb

```diff
@@ -2,20 +2,34 @@
 
 module LocalLinksManager
   class DistributedLock
+    attr_accessor :lock_name
+
+    APP = "local-links-manager"
     LIFETIME = (60 * 60) # seconds
 
     def initialize(lock_name)
       @lock_name = lock_name
     end
 
     def lock(lock_obtained:, lock_not_obtained:)
-      Services.redis.lock("local-links-manager:#{Rails.env}:#{@lock_name}", life: LIFETIME) do
+      Services.redis.lock("#{APP}:#{lock_name}", life: LIFETIME, owner: "#{APP}") do
         Rails.logger.debug('Successfully got a lock. Running...')
         lock_obtained.call
       end
     rescue Redis::Lock::LockNotAcquired => e
-      Rails.logger.debug("Failed to get lock for #{@lock_name} (#{e.message}). Another process probably got there first.")
+      Rails.logger.debug("Failed to get lock for #{lock_name} (#{e.message}). Another process probably got there first.")
       lock_not_obtained.call
     end
+
+    def unlock
+      Redis::Lock.new(Services.redis, "#{APP}:#{lock_name}", owner: "#{APP}").unlock
+      Rails.logger.debug("Successfully unlocked #{lock_name}")
+    rescue StandardError => e
+      Rails.logger.error("Failed to unlock #{lock_name}\n#{e.message}")
+    end
+
+    def locked?
+      Redis::Lock.new(Services.redis, "#{APP}:#{lock_name}").locked?
```

Comment:
Not sure how `Redis::Lock` works, but will this need the `owner: APP` argument added?  If so, maybe we can move that into a helper method so that all the places that need to build a `Redis::Lock` do the same thing.